packages/storybook/stories/va-text-input.stories.jsx

```diff
@@ -22,9 +22,9 @@ const defaultArgs = {
   'autocomplete': false,
   'enable-analytics': false,
   'required': false,
-  'error': null,
-  'maxlength': null,
-  'value': null,
+  'error': undefined,
+  'maxlength': undefined,
+  'value': undefined,
```

Comment:
These were updated to `undefined` to prevent them from showing up in Storybook's `Show Code` as `error={null}`.